File: Easy/Python/1752. Check if Array Is Sorted and Rotated.py
class Solution(object):
    def check(self, nums):
        count = 0
        n = len(nums)

        for i in range(1, n):
            if nums[i] < nums[i - 1]:
                count += 1
        
        if 1 < count or (count != 0 and nums[0] < nums[n - 1]):
            return False
        return True

File: Easy/Java/1752. Check if Array Is Sorted and Rotated.java
class Solution {
    public boolean check(int[] nums) {
        int count = 0;
        int n = nums.length;

        for(int i = 1; i < n; i++){
            if(nums[i] < nums[i - 1])
                count++;
        }

        if(1 < count || (count != 0 && nums[0] < nums[n - 1]))
            return false;
        return true;
    }
}
